sugoi-api-jms-store-provider/src/main/java/fr/insee/sugoi/jms/JmsWriterStore.java

```diff
@@ -370,4 +370,16 @@ private ProviderResponse checkAndSend(
 
     return response;
   }
+
+  private RuntimeException createExceptionFromResponse(ProviderResponse providerResponse) {
+    try {
+      Class<?> c = Class.forName(providerResponse.getExceptionType());
+      return (RuntimeException)
+          c.getDeclaredConstructor(String.class, Throwable.class)
+              .newInstance(
+                  providerResponse.getException().getMessage(), providerResponse.getException());
+    } catch (Exception e) {
+      return new RuntimeException(e.toString());
```

Comment:
maybe return a new runtimeException(providerresponse.exception.message,providerresponse.exception)
?

Reply:
yes i do it